src/pe_reports/pages.py

```diff
@@ -1,213 +1,313 @@
+    # Build tables
+    risky_assets = risky_assets[:7]
+    risky_assets.columns = ["IP", "Protocol"]
+    risky_assets_table = buildTable(risky_assets, ["table"], [50, 50])
+    verif_vulns.columns = ["CVE", "IP", "Port"]
+    verif_vulns_table = buildTable(verif_vulns, ["table"], [40, 40, 20])
+    verif_vulns_summary_table = buildTable(
+        verif_vulns_summary, ["table"], [15, 15, 15, 55]
+    )
+
+    # Update chevrion dictionary
+    vulns_dict = {
+        "verif_vulns": verif_vulns_table,
+        "verif_vulns_summary": verif_vulns_summary_table,
+        "risky_assets": risky_assets_table,
+        "riskyPorts": risky_ports_count,
+        "verifVulns": total_verif_vulns,
+        "unverifVulns": unverified_vuln_count,
+    }
+    chevron_dict.update(vulns_dict)
+    return (
+        chevron_dict,
+        Malware_Vuln.insecure_df,
+        Malware_Vuln.vulns_df,
+        Malware_Vuln.assets_df,
+    )
+
+
+def dark_web(chevron_dict, start_date, end_date, org_uid):
+    """Page 6: Web & Dark Web Mentions."""
+    Cyber6 = Cyber_Six(start_date, end_date, org_uid)
+    dark_web_count = Cyber6.dark_web_count()
+    dark_web_date = Cyber6.dark_web_date()
+    dark_web_sites = Cyber6.dark_web_sites()
+    alert_threats = Cyber6.alerts_threats()
+    dark_web_bad_actors = Cyber6.dark_web_bad_actors()
+    dark_web_tags = Cyber6.dark_web_tags()
+    dark_web_content = Cyber6.dark_web_content()
+    alert_exec = Cyber6.alerts_exec()
+    dark_web_most_act = Cyber6.dark_web_most_act()
+    alerts_site = Cyber6.alerts_site()
+    top_cve_table = Cyber6.top_cve_table()
+    # Build dark web mentions over time line chart
+    width = 19
+    height = 9
+    name = "web_only_df_2"
+    title = ""
+    x_label = "Dark Web Mentions"
+    y_label = "Mentions count"
+    dark_mentions_chart = Charts(
+        dark_web_date,
+        width,
+        height,
+        name,
+        title,
+        x_label,
+        y_label,
+    )
+    dark_mentions_chart.line_chart()
+    # Build forum type / conversation content pie chart
+    width = 19
+    height = 9
+    name = "dark_web_forum_pie"
+    title = ""
+    x_label = ""
+    y_label = ""
+    pie_chart = Charts(
+        dark_web_content,
+        width,
+        height,
+        name,
+        title,
+        x_label,
+        y_label,
+    )
+    pie_chart.pie()
+
+    # Build tables
+    dark_web_sites_table = buildTable(dark_web_sites, ["table"], [50, 50])
+    alerts_threats_table = buildTable(alert_threats, ["table"], [40, 40, 20])
+    dark_web_actors_table = buildTable(dark_web_bad_actors[:10], ["table"], [50, 50])
+    dark_web_tags_table = buildTable(dark_web_tags, ["table"], [60, 40])
+    alerts_exec_table = buildTable(alert_exec[:8], ["table"], [15, 70, 15])
+    dark_web_act_table = buildTable(dark_web_most_act, ["table"], [75, 25])
+    alerts_site_table = buildTable(alerts_site, ["table"], [50, 50])
+    top_cves_table = buildTable(top_cve_table, ["table"], [30, 70])
```

Comment:
Addressed [here](https://github.com/cisagov/pe-reports/pull/114/commits/82e1864bcadcc388b28b84ed41537ca6fd8b4bfc). I explained the significance of the 3 argument in the buildTable function instead of here. Let me know if that's ok.

Reply:
That works, thanks.